irohad/peer_service/peer_service.cpp

```diff
@@ -14,45 +14,74 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <numeric>
+#include <random>
+#include <unordered_set>
+#include <exception>
+
 #include <peer_service/peer_service.hpp>
 
-// TODO
 namespace peer_service {
 
   void initialize() {
```

Comment:
This is like class constructor, but without context of the class it look like calling some random method 'initialise()' from some other class. Much clear will be using  notation like 'PeerService peer_service()' instead of initialise method. A pleasant bonus for that, each object has also destructor and memory will be cleared after.